### PR TITLE
feat(ui): Select supports primary prop

### PR DESCRIPTION
**Pull request by beausmith**
image

### CODE REVIEW

**Comment by beausmith** [comment]
These tests are passing locally. May have to "git clean -dfX"…

**Comment by jonahkagan** [comment]
Could you add a screenshot of what this change looks like?

**Comment by beausmith** [comment]
@jonahkagan - updated!

File: frontends/bas/src/App.css [hunk 1]
```diff
@@ -31,6 +31,14 @@ legend {
   display: block;
 }
 
+select option {
+  background-color: #ffffff;
+  color: #000000;
+  &:disabled {
+    color: #999999;
+  }
+}
```
Comment (jonahkagan) [inline frontends/bas/src/App.css:40]: What are these styles for?

**Review by jonahkagan — APPROVED** [review]

File: frontends/bas/src/App.css (hunk 1 above)
Comment (beausmith) [reply frontends/bas/src/App.css:40]: These styles ensure that when the options are displayed, the background is always white, the disabled options are grey, and the enabled options are black. 

When adding the primary style, the kiosk browser version displayed their background as green when the Select was primary. 🤦

Comment (jonahkagan) [reply frontends/bas/src/App.css:40]: Yikes! Is there any reason to define them here vs in the Select component? Seems like splitting them up means you have to look in multiple places to understand the styles, but maybe you have a reason?

Comment (beausmith) [reply frontends/bas/src/App.css:40]: Thanks a good point/question. I appreciate your questions. I guess I consider this a global reset that I want to apply everywhere a select is used. If a new Component is created which contains a <select> tag, I want that select tag to have these styles too. However, it's mostly the white background that I want to ensure is consistent across browsers. The definition of grey and black text colors were perhaps an overreach and can be moved to the Select component. I'll fix this. Please LMK if I can clarify anything further.

Comment (jonahkagan) [reply frontends/bas/src/App.css:40]: That makes sense - I had a feeling this file was a "reset" file. Sounds good to me!

Comment (beausmith) [reply frontends/bas/src/App.css:40]: I take it back; these do need to be "reset" css as if they are not specified, the `<option>` element styles inherit from the parent `<select>` element when they should not (see below). This reset can be easily overridden in a styled-component. TL;DR: I'm going to leave these styles in the reset css files for each app.
https://user-images.githubusercontent.com/28444/190001086-5e1b7853-d0cf-4ade-ad5c-ac966da376ae.mov

Comment (beausmith) [reply frontends/bas/src/App.css:40]: Bummed that these videos don't display inline as they do in the above description. 🤷‍♂️

Comment (jonahkagan) [reply frontends/bas/src/App.css:40]: Sounds good!

**Pull request merged**